docs/src/main/sphinx/installation/cli.rst

```diff
@@ -334,7 +336,7 @@ with a wrapper script.
       --catalog <catalog> \
       --schema <schema>
 
-The following table list the available options for Kerberos authentication:
+The following table lists the available options for Kerberos authentication:
```

Comment:
You can possibly cross-ref `Kerberos authentication` to :doc:`/security/kerberos`, but that reference is already on line 318, so maybe this would be overkill? Get a consensus on tis.